test/retry-handler.js

```diff
@@ -695,8 +695,15 @@ test('Should handle 206 partial content - bad-etag', async t => {
             t.ifError('should not complete')
           },
           onError (err) {
-            t.strictEqual(Buffer.concat(chunks).toString('utf-8'), 'abc')
-            t.strictEqual(err.code, 'UND_ERR_REQ_RETRY')
+            try {
+              t.strictEqual(Buffer.concat(chunks).toString('utf-8'), 'abc')
+              t.strictEqual(err.code, 'UND_ERR_REQ_RETRY')
+              t.strictEqual(err.message, 'ETag mismatch')
+              t.deepEqual(err.data, { count: 2 })
+            } catch (e) {
```

Comment:
if any assertion in `onError` fails, the test hangs and then times out without any feedback. Maybe I'm doing something wrong, but it was hard to debug.

Reply:
Is there a better way to plan? Would you like me to remove the try/catch?